Ports/SuperTuxKart/patches/0004-Remove-IPv6-support.patch

```diff
@@ -0,0 +1,904 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Dominika Liberda <ja@sdomi.pl>
+Date: Sun, 2 Jul 2023 06:13:05 +0200
+Subject: [PATCH] Remove IPv6 support
+
+---
+ CMakeLists.txt                              |   6 +-
+ lib/enet/unix.c                             |   5 +-
+ src/network/network_config.cpp              |  24 +-
+ src/network/network_console.cpp             |   1 +
+ src/network/protocols/connect_to_server.cpp |  22 +-
+ src/network/protocols/server_lobby.cpp      |  17 +-
+ src/network/servers_manager.cpp             |   8 +-
+ src/network/socket_address.cpp              |  57 +--
+ src/network/stk_host.cpp                    |   4 +-
+ src/network/stk_ipv6.cpp                    | 514 --------------------
+ 10 files changed, 29 insertions(+), 629 deletions(-)
+ delete mode 100644 src/network/stk_ipv6.cpp
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 98dd5dcaef4547717f92440b269216b8994a29f0..a68f38ec32e7d240dae28998caf5511e936533ea 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -231,9 +231,9 @@ if((UNIX AND NOT APPLE) AND USE_SYSTEM_ENET AND NOT USE_IPV6)
+     pkg_check_modules(ENET libenet>=1.3.4)
+ endif()
+ 
+-if (USE_IPV6)
+-   add_definitions(-DENABLE_IPV6)
+-endif()
++#if (USE_IPV6)
++#   add_definitions(-DENABLE_IPV6)
```

Comment:
USE_IPV6 is a CMake option. Add turning it off as a configopts setting rather than commenting out things here

```
CMAKE_DEPENDENT_OPTION(USE_IPV6 "Allow create or connect to game server with IPv6 address, system enet will not be used." ON
  "NOT USE_SWITCH" OFF)
```

Reply:
I suspect that setting -DUSE_IPV6=OFF will make a bunch of this patch redundant as the ipv6 related files will not be compiled at all.

Reply:
> ipv6 related files will not be compiled at all.

That's unfortunately not the case, the IPv6-less build seems hopelessly broken